neo4j/README.md

```diff
@@ -1,64 +1,61 @@
-# Neo4j Integration
+# Agent Check: neo4j
 
 ## Overview
 
-Get metrics from Neo4j service in real time to:
-
-- Visualize and monitor Neo4j states.
-- Be notified about Neo4j failovers and events.
+This check monitors [neo4j][1] through the Datadog Agent.
+Please verify the metrics and checks that are submitted through this agent. Since Neo4j 4.0 and onward supports multiple databases, some metrics and checks are no longer published.
```

Comment:
```suggestion
Verify the metrics and checks that are submitted through this Agent. Because Neo4j 4.0 and onward supports multiple databases, some metrics and checks are no longer published.
```
What does it mean to "verify" the metrics and checks? Why are we telling users to do this? I can't tell what it means, just that it's a very unusual statement in documentation.